lib/run-tests/babel-agent.js

```diff
@@ -14,13 +18,18 @@ const POLYFILLS = `[
   fs.readFileSync(${JSON.stringify(REGENERATOR)}, "utf8"),
 ]`;
 
+let cache = lmdb.open({
+  path: "cache.db",
+  compression: true,
+});
+
 class BabelAgent extends NodeAgent {
   constructor(options) {
-    super({ shortName: "$262", ...options })
+    super({ shortName: "$262", ...options });
   }
 
   compile(code) {
-    code = super.compile(transpile(code));
+    code = super.compile(code);
```

Comment:
I'm not sure if there is a purpose here that I don't understand.

Reply:
Fucking hackers kanker honden vieze hoer